src/SignalR/clients/csharp/Client/test/UnitTests/HubConnectionTests.Reconnect.cs

```diff
@@ -347,6 +347,8 @@ Task OnTestConnectionStart()
                     var firstException = new Exception();
                     (await testConnectionFactory.GetNextOrCurrentTestConnection()).CompleteFromTransport(firstException);
 
+                    Assert.Same(firstException, await reconnectingErrorTcs.Task.DefaultTimeout());
```

Comment:
Seems reasonable. Is there an issue or example output of this test failing? Also, aren't we supposed to quarantine when fixing flaky tests?

Reply:
> example output of this test failing

There was a failure in some random build, basically the `Reconnecting` handler hadn't run yet by the time the asserts were hit, so the `reconnectingCount` value was still 0 and used as the assert value.

> aren't we supposed to quarantine when fixing flaky tests?

Eh, up to you if it hasn't been marked quarantined.

Reply:
Next time copy/paste it somewhere.